src/arrayMethodSort.js

```diff
@@ -5,7 +5,24 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    for (let i = 0; i < this.length; i++) {
+      for (let j = 0; j < this.length - 1; j++) {
+        const current = this[j];
+        const next = this[j + 1];
+
+        if (compareFunction === undefined) {
```

Comment:
Create a variable that will hold the result of comparing, f.e. `isCurrentBigger` instead of `if...else` construction.
Use ternary.